charts/argo-cd/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 2.0.5
 description: A Helm chart for ArgoCD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 3.11.4
+version: 3.11.5
```

Comment:
Unfortunately, the changelog annoation was not updated... 
Friendly reminder: Please keep changelog annoation update in mind before merge @mbevc1 :-)

Reply:
Ah, sorry totally missed that after last rebase as there were issues with signing commits.